content/en/blog/_posts/2016-12-00-Kubernetes-1-5-Supporting-Production-Workloads.md

```diff
@@ -58,9 +58,11 @@ Offline, the community can be found at one of the many Kubernetes related [meetu
 
 
 
+
 Ready to start contributing? Share your voice at our weekly [community meeting](https://kubernetes.io/community/).&nbsp;  
 
 
+
 - Post questions (or answer questions) on [Stack Overflow](http://stackoverflow.com/questions/tagged/kubernetes)&nbsp;
```

Comment:
I recommend making no changes to this file; it's good as-is.

Reply:
@sftim I have deleted unused null line.